index.html

```diff
@@ -189,10 +223,12 @@ <h3 class="white text-center"><i class="icon fa  fa-spinner"></i></h3>
 
             </div>
             <!-- Default to the left -->
-            <strong>Copyright &copy; 2015 <a target="_blank" href="http://ehealthafrica.org">eHealth Africa</a>.</strong> All rights reserved.
+            <strong>Copyright &copy; 2015 <a href="#">eHealth Africa</a>.</strong> All rights reserved.
```

Comment:
Add a target="_blank" to this hyperlink and also full url to ehealth africas website in the href attribute